src/settings.ml

```diff
@@ -113,3 +113,17 @@ let server_args_setting =
     ~key:"ocaml.server.args"
     ~of_json:Jsonoo.Decode.(list string)
     ~to_json:Jsonoo.Encode.(list string)
+
+let server_codelens_setting =
+  create_setting
+    ~scope:ConfigurationTarget.Workspace
+    ~key:"ocaml.server.codelens"
+    ~of_json:Jsonoo.Decode.(bool)
+    ~to_json:Jsonoo.Encode.(bool)
+
+let server_extendedHover_setting =
+  create_setting
+    ~scope:ConfigurationTarget.Workspace
+    ~key:"ocaml.server.extendedHover"
+    ~of_json:Jsonoo.Decode.(bool)
+    ~to_json:Jsonoo.Encode.(bool)
```

Comment:
The parentheses around `bool` aren't necessary.

Reply:
right, fixed